{"level":"info","datetime":"2021-04-03T16:40:57.902Z","logger":"main","caller":"logs/log.go:39","msg":"Server is running","type":"process","error":null}
{"level":"error","datetime":"2021-04-03T16:40:57.904Z","logger":"main.Postgres","caller":"logs/log.go:44","msg":"Try to ping postgres","type":"process","error":"dial tcp 172.20.0.2:5432: connect: connection refused","stacktrace":"github.com/manabie-com/togo/internal/logs.(*Logger).Error\n\t/app/internal/logs/log.go:44\ngithub.com/manabie-com/togo/internal/storages/postgres.(*Postgres).pingDB\n\t/app/internal/storages/postgres/db.go:77\ngithub.com/manabie-com/togo/internal/storages/postgres.NewPostgres\n\t/app/internal/storages/postgres/db.go:45\nmain.main\n\t/app/main.go:30\nruntime.main\n\t/usr/local/go/src/runtime/proc.go:203"}
